app/activeproject.cpp

```diff
@@ -153,9 +153,16 @@ bool ActiveProject::forceLoad( const QString &filePath, bool force )
       CoreUtils::log( QStringLiteral( "Project loading" ), QStringLiteral( "Could not read project file: " ) + error );
 
       mLocalProject = LocalProject();
-      mQgsProject->clear();
-      QList< QgsMapLayer * > layers;
-      mMapSettings->setLayers( layers );
+      if ( mQgsProject )
```

Comment:
Hmm, I think there is no point in checking if `mQgsProject` is valid here. It is accessed directly a few lines above.

Reply:
It should always be valid as we assign the QGIS singleton to it in the constructor